plugins/org.jkiss.dbeaver.ext.vertica/src/org/jkiss/dbeaver/ext/vertica/model/VerticaDataSource.java

```diff
@@ -146,6 +147,14 @@ protected Connection openConnection(
                  warning = warning.getNextWarning()
             ) {
                 if (checkForPasswordWillExpireWarning(warning)) {
+                    DBAUserPasswordManager manager = getAdapter(DBAUserPasswordManager.class);
+                    if (manager != null &&
+                        DBWorkbench.getPlatformUI().confirmAction(
+                            "Change password now",
+                            "Would you like to change your password now?")
```

Comment:
```suggestion
                            "Your password is about to expire. Would you like to change your password now?")
```
And there is no localization

Reply:
This dialog is right after this one

![2023-03-02 20_19_56-DBeaver Ultimate 23 0 0 - film](https://user-images.githubusercontent.com/45152336/222503961-69199971-a370-41a6-bfc9-2d1ec27924da.png)

Reply:
Fine. That's what I was asking about.